metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -3457,33 +3493,88 @@ def _create_workbook_properties(
         return {"permissions": json.dumps(groups)} if len(groups) > 0 else None
 
     def ingest_tableau_site(self):
+        self.report.report_ingestion_stage_start(
```

Comment:
eventually I want to make these stages context managers - explicitly reporting start/end feels pretty error-prone